Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM tensorflow/tensorflow:1.0.1
```

Comment:
I've tried updating to 1.1 but this results in a runtime error when importing sonnet

import sonnet as snt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/sonnet/__init__.py", line 102, in <module>
    from sonnet.python.ops.resampler import resampler
  File "/usr/local/lib/python2.7/dist-packages/sonnet/python/ops/resampler.py", line 33, in <module>
    tf.resource_loader.get_path_to_datafile("_resampler.so"))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/load_library.py", line 64, in load_op_library
    None, None, error_msg, error_code)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/local/lib/python2.7/dist-packages/sonnet/python/ops/_resampler.so: undefined symbol: _ZN10tensorflow8internal21CheckOpMessageBuilder9NewStringB5cxx11Ev

updating to 1.1 would remove the need to import the ppa:openjdk-r/ppa repo

Happy to update this patch if a workaround for that can be found.